### ISSUE TITLE
 IPython console seems to not connect the kernel

### ISSUE THREAD

**Issue by mguaita** [spyder-ide/spyder#21232]
### What steps will reproduce the problem?

Open anaconda navigator, launch spyder.
The IPython console seems to not connect the kernel.

### Traceback
```python-traceback
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/zmq/eventloop/zmqstream.py", line 584, in _run_callback
    f = callback(*args, **kwargs)
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/jupyter_client/threaded.py", line 123, in _handle_recv
    msg_list = self.ioloop._asyncio_event_loop.run_until_complete(get_msg(future_msg))
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/nest_asyncio.py", line 90, in run_until_complete
    return f.result()
  File "/home/adminiis/anaconda3/lib/python3.8/asyncio/futures.py", line 178, in result
    raise self._exception
  File "/home/adminiis/anaconda3/lib/python3.8/asyncio/tasks.py", line 280, in __step
    result = coro.send(None)
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/jupyter_client/threaded.py", line 34, in get_msg
    return await msg
TypeError: object list can't be used in 'await' expression
ERROR:tornado.general:Uncaught exception in zmqstream callback
Traceback (most recent call last):
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/zmq/eventloop/zmqstream.py", line 634, in _handle_events
    self._handle_recv()
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/zmq/eventloop/zmqstream.py", line 663, in _handle_recv
    self._run_callback(callback, msg)
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/zmq/eventloop/zmqstream.py", line 584, in _run_callback
    f = callback(*args, **kwargs)
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/jupyter_client/threaded.py", line 123, in _handle_recv
    msg_list = self.ioloop._asyncio_event_loop.run_until_complete(get_msg(future_msg))
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/nest_asyncio.py", line 90, in run_until_complete
    return f.result()
  File "/home/adminiis/anaconda3/lib/python3.8/asyncio/futures.py", line 178, in result
    raise self._exception
  File "/home/adminiis/anaconda3/lib/python3.8/asyncio/tasks.py", line 280, in __step
    result = coro.send(None)
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/jupyter_client/threaded.py", line 34, in get_msg
    return await msg
TypeError: object list can't be used in 'await' expression
ERROR:asyncio:Exception in callback BaseAsyncIOLoop._handle_events(107, 1)
handle: <Handle BaseAsyncIOLoop._handle_events(107, 1)>
Traceback (most recent call last):
  File "/home/adminiis/anaconda3/lib/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/tornado/platform/asyncio.py", line 189, in _handle_events
    handler_func(fileobj, events)
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/zmq/eventloop/zmqstream.py", line 634, in _handle_events
    self._handle_recv()
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/zmq/eventloop/zmqstream.py", line 663, in _handle_recv
    self._run_callback(callback, msg)
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/zmq/eventloop/zmqstream.py", line 584, in _run_callback
    f = callback(*args, **kwargs)
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/jupyter_client/threaded.py", line 123, in _handle_recv
    msg_list = self.ioloop._asyncio_event_loop.run_until_complete(get_msg(future_msg))
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/nest_asyncio.py", line 90, in run_until_complete
    return f.result()
  File "/home/adminiis/anaconda3/lib/python3.8/asyncio/futures.py", line 178, in result
    raise self._exception
  File "/home/adminiis/anaconda3/lib/python3.8/asyncio/tasks.py", line 280, in __step
    result = coro.send(None)
  File "/home/adminiis/anaconda3/lib/python3.8/site-packages/jupyter_client/threaded.py", line 34, in get_msg
    return await msg
TypeError: object list can't be used in 'await' expression
```

## Versions

* Spyder version: 4.2.1 
* Python version: 3.8.6
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Linux 5.15.0-78-generic

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0            :  1.4.1 (OK)
chardet >=2.0.0                 :  5.1.0 (OK)
cloudpickle >=0.5.0             :  2.2.1 (OK)
diff_match_patch >=20181111     :  20230430 (OK)
intervaltree >=3.0.2            :  3.1.0 (OK)
IPython >=7.6.0                 :  8.12.0 (OK)
jedi =0.17.2                    :  0.17.2 (OK)
jsonschema >=3.2.0              :  4.18.0 (OK)
keyring >=17.0.0                :  24.2.0 (OK)
nbconvert >=4.0                 :  7.6.0 (OK)
numpydoc >=0.6.0                :  1.5.0 (OK)
parso =0.7.0                    :  0.7.0 (OK)
pexpect >=4.4.0                 :  4.8.0 (OK)
pickleshare >=0.4               :  0.7.5 (OK)
psutil >=5.3                    :  5.9.5 (OK)
pygments >=2.0                  :  2.15.1 (OK)
pylint >=1.0                    :  2.17.4 (OK)
pyls >=0.36.2;<1.0.0            :  0.36.2 (OK)
pyls_black >=0.4.6              :  0.4.6 (OK)
pyls_spyder >=0.3.0             :  0.3.0 (OK)
qdarkstyle >=2.8                :  3.1 (OK)
qtawesome >=0.5.7               :  1.2.3 (OK)
qtconsole >=5.0.1               :  5.4.3 (OK)
qtpy >=1.5.0                    :  2.3.1 (OK)
rtree >=0.8.3                   :  1.0.1 (OK)
setuptools >=39.0.0             :  59.8.0 (OK)
sphinx >=0.6.6                  :  7.0.1 (OK)
spyder_kernels >=1.10.1;<1.11.0 :  1.10.1 (OK)
textdistance >=4.2.0            :  4.5.0 (OK)
three_merge >=0.1.1             :  0.1.1 (OK)
watchdog >=0.10.3               :  3.0.0 (OK)
xdg >=0.26                      :  6.0.0 (OK)
zmq >=17                        :  25.1.0 (OK)

# Optional:
cython >=0.21                   :  0.29.36 (OK)
matplotlib >=2.0.0              :  3.5.3 (OK)
numpy >=1.7                     :  1.20.3 (OK)
pandas >=1.1.1                  :  1.1.3 (OK)
scipy >=0.17.0                  :  1.5.3 (OK)
sympy >=0.7.3                   :  1.12 (OK)
```

**Comment by ccordoba12** [comment]
Hey @mguaita, thanks for reporting. This problem is fixed in our latest version available in Anaconda (5.4.3). Please update by closing Spyder, opening the Anaconda Prompt and running there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).

Let us know if that works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
